index.html

```diff
@@ -3,14 +3,113 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name = "viewport" content = "width=device=width, intial=scale=1">
   <title>Great Idea - Responsive I</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
 </head>
 
-<body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+<body>  
+  <div class = "container">
+    <header>
+  <nav>
+<a href ="index.html">Services</a>
+<a href ="#">Product</a>
+    <a href ="#">Vision</a>
+        <a href ="#">Features</a>
+            <a href ="#">About</a>
+                <a href ="#">Contact</a>
+</nav>
+
+<img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+</header>
+
+<section class="sub">
+
+<h1>Innovation<br>
+On <br>
+Demand</h1> <br>
+
+<h3>Get Started</h3>
+
+<img class = "codepic" src="img/header-img.png" alt="Image of a code snippet.">
+
+<div class = "line">
+
+</section> 
+
+<div  class = "one">
+<h6>Features</h6>
+<div class = featuresp>
+<p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+<br>
+</div>
+
+<h3>About</h3>
+</div>
+<div class = "two">
+<div class = "aboutp">
+<p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+<br>
+<br>
+<br>
+<br>
+<img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+<br>
+<br>
+</div>
+<div class = "three">
+
+  <h4>Services</h4>
+  <div class = "servicesp">
+<p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+
+<h5>Product</h5>
+
+<div class = "four">
+  <div class = "productp">
+<p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</div>
+</div>
+
+<div class = "five">
+
+<h3>Vision</h3>
+
+<p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+<br>
+
+<br>
+
+<div class = "line2"></div>
+   
+<div class = "contact-section">
+<h2>Contact</h2>
+<address>
+<br>
+123 Way 456 Street <br>
+Somewhere, USA
+<br>
+<br>
+1 (888) 888-8888
+<br>
+<br>
+sales@greatidea.io
+<br>
+<br>
+</address>
+</section>
+<br>
+<br>
```

Comment:
Remember what I said about using margins and padding whenever you can over `<br>` tags. It makes it easier to maintain down the line!